services/blockchain-indexer/shared/indexer/utils/account.js

```diff
@@ -81,5 +101,6 @@ module.exports = {
 	getLisk32Address,
 	updateAccountPublicKey,
 	getHexAddress,
-	getAccountsTable,
+	getAccountsTableReplica,
```

Comment:
Please do not rename the method. Rather move it to the dataService layer as discussed.

Reply:
Applied.